pygetty/formatters.py

```
@@ -67,12 +67,8 @@ def format_video(video):
         video['keywords'] = [kw['text'] for kw in video['keywords']]
 
     if video.get('mastered_to') is not None:
-        video['parsed_dimensions'] = {
-            k: int(v)
-            for k, v in re.search(
-                pattern=MASTERY_DIMENSIONS_REGEX,
-                string=video['mastered_to'],
-            ).groupdict().viewitems()
-        }
+        matches = re.search(pattern=MASTERY_DIMENSIONS_REGEX, string=video['mastered_to'])
+        if matches is not None:
+            video['parsed_dimensions'] = {k: int(v) for k, v in matches.groupdict().viewitems()}
```

@dacox currently it looks like downstream we assume `parsed_dimensions` is set.  If we don't get a match, should we just throw an exception?  Or look somewhere else?  Or update the downstream to ignore the result?

---

We'll get an exception either way; the location of the exception is all that changes. Here we'll get a `NoneType has no attribute groupdict` or something to that effect, with this change we'll push it downstream to become a `KeyError` (which we can talk about in that repo - reminder that `pygetty` is publicly accessible :) )